_docs/geyser/api.md

```diff
@@ -81,6 +81,14 @@ like currently, showing emotes.
 The item package contains classes and interfaces related to items in Geyser. You can create custom items and register them using the Geyser API.
 See [here](/geyser/custom-items) for detailed information on how to register custom items.
 
+#### [Block](https://github.com/GeyserMC/Geyser/tree/master/api/src/main/java/org/geysermc/geyser/api/block)
+The block package contains classes and interfaces related to blocks in Geyser. You can create custom blocks, and register them in using the [GeyserDefineCustomBlocksEvent](https://github.com/GeyserMC/Geyser/blob/master/api/src/main/java/org/geysermc/geyser/api/event/lifecycle/GeyserDefineCustomBlocksEvent.java).
+See [here](/geyser/custom-blocks) for detailed information.
+
+#### Skull
```

Comment:
Sorry, my bad - just checked, and skulls are registered in a separate event, so no need to mention them here